server/src/main/java/org/candlepin/jackson/DateSerializer.java

```diff
@@ -24,26 +24,27 @@
 import java.util.Date;
 import java.util.TimeZone;
 
-
-
 /**
  * DateSerializer
- *
- * This serializer removes milliseconds from Date objects in order to be more compatible with MySql.
+ * This serializer removes milliseconds from Date objects in order to be more
+ * compatible with MySql.
  * The format used is ISO 8601
  */
 public class DateSerializer extends JsonSerializer<Date> {
+
     // This SimpleDateFormat is iso 8601 without milliseconds
     public static final SimpleDateFormat ISO_8601_WITHOUT_MILLISECONDS;
 
     static {
-        ISO_8601_WITHOUT_MILLISECONDS = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+        ISO_8601_WITHOUT_MILLISECONDS = new SimpleDateFormat(
```

Comment:
We have a line-limit of 110 characters, which is preferred to lower limits of 80, as lower limits encourage lots of hanging lines and line breaks, which itself can lead to refactor issues like #1315, and is generally uglier and harder to read.

I'm not going to NACK for this, but these type of things are slowly being undone as they're encountered in the codebase, not encouraged.